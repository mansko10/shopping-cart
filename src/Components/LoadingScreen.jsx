import StyledLoader from "./Styles/StyledLoader.styled";

export default function LoadingScreen() {
  return (
    <StyledLoader>
      <div className="loader"></div>
    </StyledLoader>
  );
}
