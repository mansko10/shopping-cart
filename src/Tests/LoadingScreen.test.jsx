import { expect, it } from "vitest";
import { render } from "@testing-library/react";
import LoadingScreen from "../Components/LoadingScreen";

it("Renders LoadingScreen.jsx correctly", () => {
  const { container } = render(<LoadingScreen />);

  expect(container).toMatchSnapshot();
});
