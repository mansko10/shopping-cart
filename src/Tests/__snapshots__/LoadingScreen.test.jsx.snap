// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Renders LoadingScreen.jsx correctly 1`] = `
<div>
  <div
    class="sc-blHHSb jJJtUO"
  >
    <div
      class="loader"
    />
  </div>
</div>
`;
